src/bitcoin-cli.cpp

```diff
@@ -9,6 +9,7 @@
 #include "rpcprotocol.h"
 #include "util.h"
 #include "utilstrencodings.h"
+#include "init.h"
```

Comment:
Is this an accidental inclusion? After moving the license method to `util.h`, it shouldn't be necessary?